tests/benchmarks/src/main/scala/benchmarks/CompileBench.scala

```diff
@@ -91,9 +94,10 @@ object CompileBench {
             val relativePath = source.toString().stripPrefix("/")
             Input.VirtualFile(relativePath, text)
           }
-          sum += fn(inputs)
+
+          fn(inputs)
         }
       }
-    sum
+      .sum
```

Comment:
Not related to the PR, I just felt the `var` and foreach were unnecessary.

Reply:
Unless it's intentional? Feels like this would have incredibly small overhead to the benchmark compared to compilation pipeline itself

Reply:
I don't think it was intentional, the change is good